common/src/main/res/drawable/selectable_top_round_corners_dark.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ripple xmlns:android="http://schemas.android.com/apk/res/android"
+    android:color="@color/ripple_dark">
+    <item>
+        <shape android:shape="rectangle">
+            <corners android:topLeftRadius="8dp"
+                android:topRightRadius="8dp"
+                android:bottomLeftRadius="0dp"
+                android:bottomRightRadius="0dp"/>
```

Comment:
Maybe this is overkill to make the top item in a list to have rounded corners on top and square on the bottom.

Reply:
Another solution would be to place all items inside a CardView and leave the ripple square. CardView will automatically cut off the rounded corners.

This doesn't always make sense to use a CardView though, so rounded corners drawable might be fine too.

Reply:
For simplicity sake, I will remove these "top" corner drawables.  No corners will be rounded for now.